sops/alerts/backend_worker_jobs_count_running_high.adoc

```diff
@@ -0,0 +1,25 @@
+:toc:
+:toc-placement!:
+
+= ThreescaleBackendWorkerJobsCountRunningHigh
+
+toc::[]
+
+== Description
+
+* This alert will trigger if the number of `backend-worker` jobs is greater than a given threshold
+
+== Troubleshooting
+
+* Check if `backend-worker` pods might be failing (so no consuming any job from redis)
```

Comment:
```suggestion
* Check if `backend-worker` pods might be failing:
- This can lead to jobs not being consumed at all.
- If left unchecked, backend's redis could fill up
```